webapp/src/ts/services/contact-summary.service.ts

```diff
@@ -107,7 +107,7 @@ export class ContactSummaryService {
     } catch (error) {
       console.error('Configuration error in contact-summary function: ' + error);
       this.feedbackService.submit('Configuration error in contact-summary function: ' + error.message, false);
-      throw new Error('Configuration error');
+      return { errorStack: error.stack };
```

Comment:
I am hesitant to change this here because it feels like proper behavior for the service to throw an error here. Also, this service is used by more than just the contacts component (e.g. by the [Forms service](https://github.com/medic/cht-core/blob/ee6b24ae719740c8a768250ef078ccce772d10cf/webapp/src/ts/services/form.service.ts#L142)).  

The cleanest alternative I have found is to update the `contacts.effect.ts` and add this catch block to the `loadContactSummary` function (when calling `this.contactSummaryService.get`):

```js
     .catch(err => {
       this.contactsActions.setContactsLoadingSummary(false);
       this.contactsActions.updateSelectedContactSummary(err);
       throw err;
     });
```

Of course, to surface useful exception information, we do not want to be burying the original exception here in the `_get` call.  I think we should just re-throw the original error instead of throwing this `Configuration Error`...

```suggestion
      throw error;
```